### PR TITLE
Improve issue, bug, pr  and feature request templates

### PR DESCRIPTION
**Pull request by the-j0k3r**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### Description of the Change

It improves the section information explanations on what is expected from user to enter in each section and hides these explanations from the final post.
This prevents user from having to manually delete these areas or comment them out before hitting the create pull request or submit new issue buttons

### Alternate Designs

Considered not having an expanded version in bug report file, but for the sake of clarity and actually making relevant information on the final post be as concise as possible opted by the version you see.

I considered further improving wording here but am waiting for some feedback which may may end up leading to that at a later time post review.

### Why Should This Be In Core?

An improvement to documentation should be part of repo.

### Benefits

issue and feature request templates currently show extraneous instructions iin open bug reports/feature request if a user does not delete them or comment them out as I did for this PR.
The benefits are three fold.

* user doesn't have to waste time removing or commenting out any extra information. 
* Improved conciseness of the final feature request or bug report  
* readers dont have to waste time skimming or reading irrelevant information and in case of bug reports any unfolded information is available at a click.

### Possible Drawbacks

None

### Applicable Issues

* TL;DR shows relevant user submitted information only in all sections.

The benefits outlined above explain and help understand the current issues. 
However as someone who has done both feature requests and bug reports in atom project, I found the process overly convoluted of trying to follow the templates instructions and after posting I found there can be further improvements since these instructions were visible after posting.
I had to delete or comment out these instructions so the resulting report was as concise as should be.

So here is to helping others not having to worry about that and also help feature request and bug report reviewers dont have to read these extraneous portions.

### CODE REVIEW

**Comment by the-j0k3r** [comment]
@rsese over to you.

**Comment by lee-dohm** [comment]
My apologies ... I had to put out some fires today that consumed my time. This hasn't fallen off my radar and I'll look at it tomorrow.

**Comment by the-j0k3r** [comment]
Needs updating for possible Global templates usage.
